LaTeX/Makefile

```
@@ -0,0 +1,30 @@
+# LaTeX コンパイル用 Makefile
+# pLaTeX + dvipdfmx + BibLaTeX(biber)
+
+MAIN = main
+PDF = $(MAIN).pdf
+
+.PHONY: all pdf clean watch open
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)